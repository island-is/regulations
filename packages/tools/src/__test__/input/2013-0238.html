<p style="text-align: center">1. gr.</p>

<p style="text-align: justify">Svofelldar breytingar verða á 1. gr. reglugerðarinnar:</p>

<ol type="a">
    <li>
        <div style="text-align: justify">
            2. málsl. fellur brott.
        </div>
    </li>

    <li>
        <div style="text-align: justify">
            Ný málsgrein bætist við, svohljóðandi: Öllum afla, sem íslensk skip veiða úr stofnum sem að hluta eða öllu leyti halda sig í efnahagslögsögu Íslands, skal landað innanlands og hann veginn í innlendri höfn.
        </div>
    </li>
</ol>

<p style="text-align: center">2. gr.</p>

<p style="text-align: justify">Við 7. gr. reglugerðarinnar bætist ný málsgrein, sem verður 2. málsgrein, svohljóðandi: Ef óunninn afli er veginn á hafnarvog og ísaður og frágenginn til útflutnings, skal draga 15% frá vegnum afla vegna íshlutfalls enda fari hann beint í flutningsfar.</p>

<p style="text-align: center">3. gr.</p>

<p style="text-align: justify">14. gr. reglugerðarinnar orðist svo:</p>

<p style="text-align: justify">Við heilvigtun skal allur afli veginn. Heimilt er að heilvigta fisk eftir hausun og skal vigta búka og hausa sérstaklega.</p>

<p style="text-align: justify">Við úrtaksvigtun skal, í kjölfar brúttóvigtunar alls afla á hafnarvog, úrtak valið af handahófi þannig að það gefi sem réttasta mynd af aflanum og vigtun framkvæmd á þann veg að:</p>

<ol type="A">
    <li>
        <div style="text-align: justify">
            Afli í tilteknum fjölda íláta er vigtaður í samræmi við 3. gr. auglýsingar sjávarútvegs- og landbúnaðarráðuneytis um lágmarksúrtak við vigtun, eða<span lang="N">:</span>
        </div>
    </li>

    <li>
        <div style="text-align: justify">
            þegar hvert kar hefur verið brúttóvigtað skal afli í tilteknum fjölda íláta vigtaður í samræmi við 4. gr. auglýsingar ráðherra um reglur um lágmarksúrtak við vigtun<span lang="N">.</span>
        </div>
    </li>
</ol>

<p style="text-align: justify">Vigtunarleyfishafi skal ávallt beita sömu aðferð við vigtun úrtaks.</p>

<p style="text-align: justify">Vigtun afla skal framkvæmd af vigtarmanni sem hlotið hefur löggildingu til vigtunar og notuð skal löggilt vog.</p>

<p style="text-align: justify">Löggiltur vigtarmaður sem jafnframt er starfsmaður löndunarhafnar getur enn fremur annast endurvigtun afla, enda sé löggilt vog notuð. Löggiltur vigtarmaður sjálfstæðs vigtunarleyfishafa, sbr. VI. kafla, getur einnig annast endurvigtun afla hafi viðkomandi vigtunarleyfishafi annast vigtun aflans við löndun enda fari vigtun fram í húsnæði hans á löggiltri vog eða hjá aðila sem hefur leyfi til endurvigtunar.</p>

<p style="text-align: center">4. gr.</p>

<p style="text-align: justify">Við 1. mgr. 15. gr. reglugerðarinnar bætast orðin: og þegar afla er landað óslægðum.</p>

<p style="text-align: center">5. gr.</p>

<p style="text-align: justify">16. gr. reglugerðarinnar orðist svo:</p>

<p style="text-align: justify">Löggiltur vigtarmaður sem annast endurvigtun afla skal fylla út og undirrita vigtarnótu þar sem, auk þess sem segir í 9. gr., skal tilgreina eftirfarandi upplýsingar fyrir hverja tegund:</p>

<p style="text-align: justify">1. Þegar um er að ræða heilvigtun:</p>

<table border="0" cellspacing="1" cellpadding="0" width="511">
    <tbody>
        <tr>
            <td width="5%">　</td>

            <td width="5%">
                <p style="text-align: justify">a)</p>
            </td>

            <td width="91%">
                <p style="text-align: justify">Heildarþunga fisks.</p>
            </td>
        </tr>

        <tr>
            <td width="5%">　</td>

            <td valign="top" width="5%">
                <p style="text-align: justify">b)</p>
            </td>

            <td width="91%">
                <p style="text-align: justify">Reiknað hlutfall íss í afla. (Brúttóafli, þ.e. afli og ís samkvæmt vigtun á hafnarvog, mínus nettóafli, þ.e. afli án íss við endurvigtun, margfaldað með 100, deilt með brúttóafla.)</p>
            </td>
        </tr>

        <tr>
            <td width="5%">　</td>

            <td width="5%">
                <p style="text-align: justify">c)</p>
            </td>

            <td width="91%">
                <p style="text-align: justify">Niðurstöðu vigtunar á óslægðum afla eftir því sem við á, sbr. 24. gr.</p>
            </td>
        </tr>
    </tbody>
</table>

<p style="text-align: justify">2. Þegar um er að ræða úrtaksvigtun A:</p>

<table border="0" cellspacing="1" cellpadding="0" width="511">
    <tbody>
        <tr>
            <td width="5%">　</td>

            <td width="5%">
                <p style="text-align: justify">a)</p>
            </td>

            <td width="91%" colspan="2">
                <p style="text-align: justify">Fjöldi og gerð íláta sem valin eru í úrtak.</p>
            </td>
        </tr>

        <tr>
            <td width="5%">　</td>

            <td width="5%">
                <p style="text-align: justify">b)</p>
            </td>

            <td width="91%" colspan="2">
                <p style="text-align: justify">Fyrir hvert ílát sem valið er í úrtak skal tilgreina:</p>
            </td>
        </tr>

        <tr>
            <td width="5%">　</td>

            <td width="5%">　</td>

            <td width="5%">
                <p style="text-align: justify">i)</p>
            </td>

            <td width="85%">
                <p style="text-align: justify">þunga fisks í hverju íláti,</p>
            </td>
        </tr>

        <tr>
            <td width="5%">　</td>

            <td width="5%">　</td>

            <td width="5%">
                <p style="text-align: justify">ii)</p>
            </td>

            <td width="85%">
                <p style="text-align: justify">þunga íss í hverju íláti,</p>
            </td>
        </tr>

        <tr>
            <td width="5%">　</td>

            <td width="5%">　</td>

            <td width="5%">
                <p style="text-align: justify">iii)</p>
            </td>

            <td width="85%">
                <p style="text-align: justify">þunga íláts.</p>
            </td>
        </tr>
    </tbody>
</table>

<p style="text-align: center">6. gr.</p>

<p style="text-align: justify">IX. kafli reglugerðarinnar fellur brott.</p>

<p style="text-align: center">7. gr.</p>

<p style="text-align: justify">Á undan 58. gr. reglugerðarinnar kemur ný grein, 58. gr. A., svohljóðandi:</p>

<p style="text-align: justify">Sé fyrirhugað að flytja út óunninn afla til sölu erlendis sem þegar hefur verið endanlega vigtaður og skráður í aflaskráningarkerfi Fiskistofu, skal útflytjandi (umráðamaður viðkomandi afla) tryggja að áður en afli er settur um borð í flutningsfarið sé tilkynnt til Fiskistofu um þau veiðiskip sem veitt hafa umræddan afla og um útflutt aflamagn sundurliðað eftir tegundum eins nákvæmlega og unnt er.</p>

<p style="text-align: center">8. gr.</p>

<p style="text-align: justify">Við reglugerðina kemur ný grein, 63. gr. A., svohljóðandi:</p>

<p style="text-align: justify">Sigli fiskiskip með afla af miðum til sölu á markaði erlendis, skal útgerð og skipstjóri fiskiskipsins tryggja að Fiskistofu séu sendar upplýsingar um aflann eigi síðar en 24 klst. áður en lagt er af stað.</p>

<p style="text-align: justify">Fiskistofu er heimilt að veita fiskiskipum leyfi til að afla sem veiddur er úr íslenskum deilistofnum sé landað erlendis, sama gildir ef sérstaklega stendur á, t.d. vegna alvarlegrar vélarbilunar hjá skipi sem vinnur afla um borð. Innheimta skal kostnað vegna ferðar eftirlitsmanns til að fylgjast með löndun úr skipinu erlendis, samkvæmt framlögðum reikningum, auk kr. 29.000, vegna hvers starfsdags eftirlitsmanns.</p>

<p style="text-align: center">9. gr.</p>

<p style="text-align: justify">Reglugerð þessi er sett með stoð í lögum nr. 57, 3. júní 1996, um umgengni um nytjastofna sjávar, lögum nr. 151, 27. desember 1996, um fiskveiðar utan lögsögu Íslands og lögum 38, 15. maí 1990, um stjórn fiskveiða, til að öðlast gildi 1. september 2013.</p>

<p style="text-align: center"><em>Atvinnuvega- og nýsköpunarráðuneytinu, 14. mars 2013.</em></p>

<p style="text-align: center">F. h. r.</p>

<p style="text-align: center"><strong>Jóhann Guðmundsson.</strong></p>

<p style="text-align: right"><em>Arnór Snæbjörnsson.</em></p>
