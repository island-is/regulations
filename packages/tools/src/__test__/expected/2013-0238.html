<h3 class="article__title">
1.
gr.
</h3>

<p>
Svofelldar
breytingar
verða
á
1.
gr.
reglugerðarinnar:
</p>

<ol type="a">

<li>
2.
málsl.
fellur
brott.
</li>

<li>
Ný
málsgrein
bætist
við,
svohljóðandi:
Öllum
afla,
sem
íslensk
skip
veiða
úr
stofnum
sem
að
hluta
eða
öllu
leyti
halda
sig
í
efnahagslögsögu
Íslands,
skal
landað
innanlands
og
hann
veginn
í
innlendri
höfn.
</li>

</ol>

<h3 class="article__title">
2.
gr.
</h3>

<p>
Við
7.
gr.
reglugerðarinnar
bætist
ný
málsgrein,
sem
verður
2.
málsgrein,
svohljóðandi:
Ef
óunninn
afli
er
veginn
á
hafnarvog
og
ísaður
og
frágenginn
til
útflutnings,
skal
draga
15%
frá
vegnum
afla
vegna
íshlutfalls
enda
fari
hann
beint
í
flutningsfar.
</p>

<h3 class="article__title">
3.
gr.
</h3>

<p>
14.
gr.
reglugerðarinnar
orðist
svo:
</p>

<p>
Við
heilvigtun
skal
allur
afli
veginn.
Heimilt
er
að
heilvigta
fisk
eftir
hausun
og
skal
vigta
búka
og
hausa
sérstaklega.
</p>

<p>
Við
úrtaksvigtun
skal,
í
kjölfar
brúttóvigtunar
alls
afla
á
hafnarvog,
úrtak
valið
af
handahófi
þannig
að
það
gefi
sem
réttasta
mynd
af
aflanum
og
vigtun
framkvæmd
á
þann
veg
að:
</p>

<ol type="A">

<li>
Afli
í
tilteknum
fjölda
íláta
er
vigtaður
í
samræmi
við
3.
gr.
auglýsingar
sjávarútvegs-
og
landbúnaðarráðuneytis
um
lágmarksúrtak
við
vigtun,
eða:
</li>

<li>
þegar
hvert
kar
hefur
verið
brúttóvigtað
skal
afli
í
tilteknum
fjölda
íláta
vigtaður
í
samræmi
við
4.
gr.
auglýsingar
ráðherra
um
reglur
um
lágmarksúrtak
við
vigtun.
</li>

</ol>

<p>
Vigtunarleyfishafi
skal
ávallt
beita
sömu
aðferð
við
vigtun
úrtaks.
</p>

<p>
Vigtun
afla
skal
framkvæmd
af
vigtarmanni
sem
hlotið
hefur
löggildingu
til
vigtunar
og
notuð
skal
löggilt
vog.
</p>

<p>
Löggiltur
vigtarmaður
sem
jafnframt
er
starfsmaður
löndunarhafnar
getur
enn
fremur
annast
endurvigtun
afla,
enda
sé
löggilt
vog
notuð.
Löggiltur
vigtarmaður
sjálfstæðs
vigtunarleyfishafa,
sbr.
VI.
kafla,
getur
einnig
annast
endurvigtun
afla
hafi
viðkomandi
vigtunarleyfishafi
annast
vigtun
aflans
við
löndun
enda
fari
vigtun
fram
í
húsnæði
hans
á
löggiltri
vog
eða
hjá
aðila
sem
hefur
leyfi
til
endurvigtunar.
</p>

<h3 class="article__title">
4.
gr.
</h3>

<p>
Við
1.
mgr.
15.
gr.
reglugerðarinnar
bætast
orðin:
og
þegar
afla
er
landað
óslægðum.
</p>

<h3 class="article__title">
5.
gr.
</h3>

<p>
16.
gr.
reglugerðarinnar
orðist
svo:
</p>

<p>
Löggiltur
vigtarmaður
sem
annast
endurvigtun
afla
skal
fylla
út
og
undirrita
vigtarnótu
þar
sem,
auk
þess
sem
segir
í
9.
gr.,
skal
tilgreina
eftirfarandi
upplýsingar
fyrir
hverja
tegund:
</p>

<p>
1.
Þegar
um
er
að
ræða
heilvigtun:
</p>

<table class="layout layout--list">

<tbody>

<tr>

<td>
</td>

<td>
a)
</td>

<td>
Heildarþunga
fisks.
</td>

</tr>

<tr>

<td>
</td>

<td>
b)
</td>

<td>
Reiknað
hlutfall
íss
í
afla.
(Brúttóafli,
þ.e.
afli
og
ís
samkvæmt
vigtun
á
hafnarvog,
mínus
nettóafli,
þ.e.
afli
án
íss
við
endurvigtun,
margfaldað
með
100,
deilt
með
brúttóafla.)
</td>

</tr>

<tr>

<td>
</td>

<td>
c)
</td>

<td>
Niðurstöðu
vigtunar
á
óslægðum
afla
eftir
því
sem
við
á,
sbr.
24.
gr.
</td>

</tr>

</tbody>

</table>

<p>
2.
Þegar
um
er
að
ræða
úrtaksvigtun
A:
</p>

<table class="layout">

<tbody>

<tr>

<td>
</td>

<td>
a)
</td>

<td colspan="2">
Fjöldi
og
gerð
íláta
sem
valin
eru
í
úrtak.
</td>

</tr>

<tr>

<td>
</td>

<td>
b)
</td>

<td colspan="2">
Fyrir
hvert
ílát
sem
valið
er
í
úrtak
skal
tilgreina:
</td>

</tr>

<tr>

<td>
</td>

<td>
</td>

<td>
i)
</td>

<td>
þunga
fisks
í
hverju
íláti,
</td>

</tr>

<tr>

<td>
</td>

<td>
</td>

<td>
ii)
</td>

<td>
þunga
íss
í
hverju
íláti,
</td>

</tr>

<tr>

<td>
</td>

<td>
</td>

<td>
iii)
</td>

<td>
þunga
íláts.
</td>

</tr>

</tbody>

</table>

<h3 class="article__title">
6.
gr.
</h3>

<p>
IX.
kafli
reglugerðarinnar
fellur
brott.
</p>

<h3 class="article__title">
7.
gr.
</h3>

<p>
Á
undan
58.
gr.
reglugerðarinnar
kemur
ný
grein,
58.
gr.
A.,
svohljóðandi:
</p>

<p>
Sé
fyrirhugað
að
flytja
út
óunninn
afla
til
sölu
erlendis
sem
þegar
hefur
verið
endanlega
vigtaður
og
skráður
í
aflaskráningarkerfi
Fiskistofu,
skal
útflytjandi
(umráðamaður
viðkomandi
afla)
tryggja
að
áður
en
afli
er
settur
um
borð
í
flutningsfarið
sé
tilkynnt
til
Fiskistofu
um
þau
veiðiskip
sem
veitt
hafa
umræddan
afla
og
um
útflutt
aflamagn
sundurliðað
eftir
tegundum
eins
nákvæmlega
og
unnt
er.
</p>

<h3 class="article__title">
8.
gr.
</h3>

<p>
Við
reglugerðina
kemur
ný
grein,
63.
gr.
A.,
svohljóðandi:
</p>

<p>
Sigli
fiskiskip
með
afla
af
miðum
til
sölu
á
markaði
erlendis,
skal
útgerð
og
skipstjóri
fiskiskipsins
tryggja
að
Fiskistofu
séu
sendar
upplýsingar
um
aflann
eigi
síðar
en
24
klst.
áður
en
lagt
er
af
stað.
</p>

<p>
Fiskistofu
er
heimilt
að
veita
fiskiskipum
leyfi
til
að
afla
sem
veiddur
er
úr
íslenskum
deilistofnum
sé
landað
erlendis,
sama
gildir
ef
sérstaklega
stendur
á,
t.d.
vegna
alvarlegrar
vélarbilunar
hjá
skipi
sem
vinnur
afla
um
borð.
Innheimta
skal
kostnað
vegna
ferðar
eftirlitsmanns
til
að
fylgjast
með
löndun
úr
skipinu
erlendis,
samkvæmt
framlögðum
reikningum,
auk
kr.
29.000,
vegna
hvers
starfsdags
eftirlitsmanns.
</p>

<h3 class="article__title">
9.
gr.
</h3>

<p>
Reglugerð
þessi
er
sett
með
stoð
í
lögum
nr.
57,
3.
júní
1996,
um
umgengni
um
nytjastofna
sjávar,
lögum
nr.
151,
27.
desember
1996,
um
fiskveiðar
utan
lögsögu
Íslands
og
lögum
38,
15.
maí
1990,
um
stjórn
fiskveiða,
til
að
öðlast
gildi
1.
september
2013.
</p>

<p class="Dags" align="center">
<em>Atvinnuvega-
og
nýsköpunarráðuneytinu,
14.
mars
2013.</em>
</p>

<p class="FHUndirskr" align="center">
F.
h.
r.
</p>

<p align="center">
<strong>Jóhann
Guðmundsson.</strong>
</p>

<p align="right">
<em>Arnór
Snæbjörnsson.</em>
</p>
